src/routes/backingImage/DiskStateMapActions.js

```diff
@@ -6,7 +6,15 @@ const confirm = Modal.confirm
 function DiskStateMapActions({ selectedRows, deleteButtonDisabled, deleteButtonLoading, deleteDisksOnBackingImage }) {
   const deleteButtonClick = () => {
     confirm({
-      title: `Are you sure to clean up the image file from the disks (${selectedRows.map(item => item.disk).join(',')}) ?`,
+      width: 'fit-content',
+      title: (
+          <>
+            <p>Are you sure you want to clean up the image file from the disks ?</p>
```

Comment:
```suggestion
            <p>Are you sure you want to remove the image file from the disks?</p>
```

Reply:
Terminology explanation: The action "clean up" generally applies to disks. During disk cleanup, certain files are removed from the target disks.

Reply:
thanks for comment, will update to use term `remove` in confirmation modal.

Reply:
The modal is referring to one specific image file that has copies on multiple disks. Because of this, the string should say *file*, not *files*.

Reply:
@jillian-maroket , I revert the files plural wording. Please have a look again. thanks